src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function conversion(value) {
```

Comment:
The conversion function could have a more descriptive name, such as 'convertCurrencyToNumber'.